examples/plotting/polar.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows hot to use polar coordinates to generate a
+grid of polar plots of rose/rohdonea curves.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot
+    :refs: :ref:`ug_topics_contour` > :ref:`ug_topics_contour_polar`
+    :keywords: polar coordinates, rose curves, rohdonea curves
```

Comment:
I've never heard the term "rohdonea" before. However, if it is common in some places that I am unaware, then this suggestion can be ignored. 
```suggestion
    :keywords: polar coordinates, rose curves
```

Reply:
> I've never heard the term "rohdonea" before. However, if it is common in some places that I am unaware, then this suggestion can be ignored.

Me neither but I saw the [wiki](https://en.wikipedia.org/wiki/Rose_(mathematics)) article on it  and took it from there.

Reply:
> I've never heard the term "rohdonea" before. However, if it is common in some places that I am unaware, then this suggestion can be ignored.

I took it from the  [wiki](https://en.wikipedia.org/wiki/Rose_(mathematics)) article

Reply:
It seems a bit obscure I would suggest removing it